### PR TITLE
add package for pylibmc, adding cyrus_sasl to libmemcached's propagatedB...

### PR DESCRIPTION
**Pull request by risicle**
...uildInputs while we're at it

### CODE REVIEW

**Comment by risicle** [comment]
Hmm could someone with more experience of the travis system tell me what's failing here?

File: pkgs/development/libraries/libmemcached/default.nix [hunk 1]
```diff
@@ -15,6 +15,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ cyrus_sasl libevent ];
 
+  propagatedBuildInputs = [ cyrus_sasl ];
```
Comment (domenkozar) [inline pkgs/development/libraries/libmemcached/default.nix]: It should then be removed from `buildInputs`

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -9031,6 +9031,26 @@ let
...
+      sha256 = "a00e75f5c76e573e95ac5b29cb8a816728930f29332c49c74dbfd326d598088f";
+      md5 = "eaed0ba707abc0eabb1356310febada1";
+    };
+
+    buildInputs = with self; [ pkgs.libmemcached pkgs.zlib ];
+
+    meta = {
+      description = "Quick and small memcached client for Python";
+      homepage = http://sendapatch.se/projects/pylibmc/;
+      license = licenses.bsd3;
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: You'll need to add `with stdenv.lib;`

File: pkgs/development/libraries/libmemcached/default.nix (hunk 1 above)
Comment (risicle) [inline pkgs/development/libraries/libmemcached/default.nix]: Ahhh I must be going blind.

**Comment by risicle** [comment]
mmm...?

It can be automatically merged, but the merge failed?

...in a file I didn't touch? (pkgs/tools/misc/cpulimit/default.nix)

**Comment by risicle** [comment]
Travis is sucking at the moment (download speeds of 17KBps?) so this may pass if someone retries it.

**Comment by nbp** [comment]
cc @domenkozar

**Comment by domenkozar** [comment]
My bad @risicle can you rebase on top of master?

**Comment by risicle** [comment]
^Travis log length exceeded and I don't currently have access to a machine that can do a local build.

**Comment by risicle** [comment]
ok, ^a nox build for this works for me locally (though i bet travis won't work)

**Comment by risicle** [comment]
@domenkozar poke

**Comment by 7c6f434c** [comment]
I don't understand what has happened here, but as of 16fff5e the master should contain all of these changes.

**Pull request closed without merging**